src/client/Microsoft.Identity.Client/Instance/Discovery/KnownMetadataProvider.cs

```diff
@@ -72,11 +72,19 @@ void AddToPublicEnvironment(InstanceDiscoveryMetadataEntry entry)
                 PreferredCache = "login-us.microsoftonline.com"
             };
 
+            InstanceDiscoveryMetadataEntry ppeCloudEntry = new InstanceDiscoveryMetadataEntry()
```

Comment:
The solution seems to be to simply add this entry to the known list. Seems like something we should have as it is well known for our first party customers? Thoughts @bgavrilMS ?

Reply:
Yes, this is correct.